src/Forms/FixCommonErrors.cs

```diff
@@ -608,16 +608,19 @@ public void FixOcrErrorsViaReplaceList(string threeLetterIsoLanguageName)
                 for (int i = 0; i < Subtitle.Paragraphs.Count; i++)
                 {
                     var p = Subtitle.Paragraphs[i];
-                    string text = ocrFixEngine.FixOcrErrors(p.Text, i, lastLine, false, OcrFixEngine.AutoGuessLevel.Cautious);
-                    lastLine = text;
-                    if (AllowFix(p, fixAction) && p.Text != text)
+                    if (AllowFix(p, fixAction))
                     {
-                        string oldText = p.Text;
-                        p.Text = text;
-                        noOfFixes++;
-                        AddFixToListView(p, fixAction, oldText, p.Text);
+                        string text = ocrFixEngine.FixOcrErrors(p.Text, i, lastLine, false, OcrFixEngine.AutoGuessLevel.Cautious);

```

Comment:
this method is very expensive and also contains `Application.DoEvents()`, so only call it when the paragraph is **allowed fix**